tests/lib.rs

```diff
@@ -732,3 +732,39 @@ fn retain() {
     vs.sort();
     assert_eq!(v, &*vs);
 }
+
+#[test]
+fn get_one_short() {
+    let x = ('x', 42);
+
+    let (r, mut w) = evmap::new();
+
+    w.insert(x.0, x);
+
+    assert_match!(r.get_one(&x.0), None);
+
+    w.refresh();
+
+    assert_match!(r.get_one(&x.0).as_deref(), Some(('x', 42)));
+}
+
+#[test]
+fn get_one_long() {
+    let x = 'x';
+
+    let (r, mut w) = evmap::new();
+
+    // Add BAG_THRESHOLD items to ensure the inner type is ValuesInner::Long.
```

Comment:
It might be good to test if the value actually got promoted to a `ValuesInner::Long` here. That may require another `#[doc(hidden)]` method.

Reply:
Haha I was also struggling with that idea and that would be a reasonable reason why :-)

Reply:
Hmm actually, since we want to test this anyway, would it be better if I structured the tests like:
```rust
// tests/lib.rs
fn test_get_one() {
  // ensure get_one gets one value from the underlying values
  // whatever that means
}
```
```rust
// src/values.rs
fn test_value_type_upgrade() {
  // ensure the value type stays short until we exceed
  // the threshold
}

fn test_get_one() {
  // ensure get_one works regardless of value type
}
```
?

Reply:
Yes, I agree with you, these should probably be unit tests instead.

Having two tests seems fine, though you'll still want to rely on the particular threshold in the larger test.

Reply:
Okay, I'll shoot for unit tests 👍

> though you'll still want to rely on the particular threshold in the larger test.

Quick question - which is "the larger test"? Sorry for all the back and forth this tiny PR has caused 😞 I'm terrified of the dollar-per-value here haha

Reply:
Oh, sorry, that was vague of me. I expect that `test_value_type_upgrade` will _just_ test that the change happens as expected at the threshold, whereas `test_get_one` will be "larger", and will also test _behavior_ after that threshold is reached.

And don't worry about it at all, I'm happy to help :)

Reply:
30f33489d114b6354da9d37bcaa2340501498b7e

Reply:
Looks great!